build/libva.spec

```diff
@@ -0,0 +1,37 @@
+Name:       libva
+Version:    %{_version}
+Release:    el7
+Summary:    Intel libva
+License:    MIT
+Source0:    %{_sourcefile}
+
+%package devel
+Group: Development/Libraries
+Summary: Development files for libva
+
+%description devel
+Development files for libva
+
+BuildRequires: automake
+BuildRequires: autoconf
+
+%description
+Intel libva
+
+%prep
+
+%setup
+
+%build
+./configure --prefix %{_prefix} --libdir %{_libdir}
+make -j12
```

Comment:
it 's better not directly use a number here. You could let script to detect the cpu core numbers

Reply:
Ok, updated to "make -j$(nproc)"